go/billing/models.py

```diff
@@ -176,6 +176,14 @@ class Meta:
         max_digits=10, decimal_places=3, default=Decimal('0.0'),
         help_text=_("The base cost per session in cents."))
 
+    session_unit_time = models.DecimalField(
+        max_digits=10, decimal_places=3, default=Decimal('20.0'),
+        help_text=_("The time of one billed unit (in seconds)."))
```

Comment:
I'm not sure I understand the problem you have. The way I assumed it would work, is that when we process a message that is an end-of-session message, we would convert the real time to an amount of session units. The cost would then be worked out by multiplying the amount of session time units by the per time unit cost. The billing statement would show the amount of session time units, the cost per time units, and the total cost.

If the session time changed at some time, it wouldn't matter because we bill in terms of session time units, instead of actual time.